src/fit.jl

```diff
@@ -85,9 +97,7 @@ function train!(estimator::LGBMEstimator, tests_names::Vector{String}, verbosity
                      "split requirements.")
         end
         if is_finished == 1
-            # save the model in serialised form, in case we should be deepcopied or serialised elsewhere
-            estimator.model = LGBM_BoosterSaveModelToString(estimator.booster, 0, 0)
```

Comment:
Please add commentary on why this was removed.

Reply:
Its the same as what happens a few lines below when we exit the function. When I first added the savemodel to string this bit me, and i copied it up there, and while developing this now I realised its much better to break form the loop and have all the end-loop logic at one place only (saves duplication too)